QuizApp/App/Screens/Quiz/QuizViewController.swift

```diff
@@ -145,4 +149,18 @@ private extension QuizViewController {
         quizCollectionView.dataSource = self
     }
 
+    func bindViewModel() {
+
+        quizViewModel
+            .$quizes
+            .sink { _ in
+                DispatchQueue.main.async {
+                    self.categoryCollectionView.reloadData()
+                    self.quizCollectionView.reloadData()
+                }
```

Comment:
properties published from VM should already be on main. Move DispatchQueue to VM